modules/ROOT/pages/dw-arrays-functions-leftjoin.adoc

```diff
@@ -0,0 +1,95 @@
+= leftJoin
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+
+
+[[leftjoin1]]
+== leftJoin&#40;Array<L&#62;, Array<R&#62;, &#40;leftValue: L&#41; &#45;&#62; String, &#40;rightValue: R&#41; &#45;&#62; String&#41;: Array<{ l: L, r?: R }&#62;
+
+Joins two array of objects by a given `ID` criteria.
+It will return an Array all the `left` items merged by `ID` with the right items in the case where it exists any.
```

Comment:
Same comment about using present instead of future tense. I would remove "criteria." Add "of" before "all." The last few words are not grammatical. Suggest rewrite:

It returns an array that contains all of the left items merged by ID with the right items. The right items that don't have IDs that match the left items are not included in the merge.